components/eam/src/physics/cam/ndrop.F90

```diff
@@ -436,6 +444,7 @@ subroutine dropmixnuc( &
    real(r8), allocatable :: coltend(:,:)       ! column tendency for diagnostic output
    real(r8), allocatable :: coltend_cw(:,:)    ! column tendency
    real(r8) :: ccn(pcols,pver,psat)    ! number conc of aerosols activated at supersat
```

Comment:
revise comment to "volumetric number conc [1/m3] of aerosols activated at supersat"